### PR TITLE
PMM-5693 Adding UI tests for Basic Annotation Checks

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: pmm-app/tests/pages/dashboardPage.js [hunk 1]
```diff
@@ -307,6 +307,25 @@ module.exports = {
     otherReportTitleWithNoData:
       "//span[contains(text(),'No Data')]//ancestor::div[contains(@class,'panel-container')]//span[contains(@class,'panel-title-text')]",
     collapsedDashboardRow: "//div[@class='dashboard-row dashboard-row--collapsed']/a",
+    annotationMarker: "(//div[contains(@class,'events_marker')])",
+  },
+
+  annotationLocator(annotationNumber) {
+    return "(//div[contains(@class,'events_marker')])[" + annotationNumber + "]";
```
Comment (atymchuk) [inline pmm-app/tests/pages/dashboardPage.js:314]: Many find that lots of quotes in one line are rather difficult to read and comprehend. I'd suggest to use template literals instead:
```suggestion
    return `(//div[contains(@class,'events_marker')])[${annotationNumber}]`;
```

File: pmm-app/tests/pages/dashboardPage.js [hunk 2]
```diff
@@ -307,6 +307,25 @@ module.exports = {
...
+  },
+
+  annotationTagText(tagValue) {
+    return "//span[contains(text(),  '" + tagValue + "')]";
```
Comment (atymchuk) [inline pmm-app/tests/pages/dashboardPage.js:318]: Same as above

File: pmm-app/tests/pages/dashboardPage.js [hunk 3]
```diff
@@ -307,6 +307,25 @@ module.exports = {
...
+  },
+
+  annotationText(annotationTitle) {
+    return "//div[contains(text(), '" + annotationTitle + "')]";
```
Comment (atymchuk) [inline pmm-app/tests/pages/dashboardPage.js:322]: Same as above

File: pmm-app/tests/pages/adminPage.js [hunk 4]
```diff
@@ -42,11 +42,12 @@ module.exports = {
     return "(//div[contains(text(), '" + dashboardName + "')])[1]";
   },
 
-  applyTimer(timeDiff) {
+  applyTimer(timeDiff = "Last 5 minutes") {
     I.click(this.fields.timePickerMenu);
     I.waitForVisible("//span[contains(text(), 'Last 5 minutes')]", 30);
-    I.click("//span[contains(text(), 'Last 5 minutes')]");
+    I.click("//span[contains(text(), '" + timeDiff + "')]");
```
Comment (atymchuk) [inline pmm-app/tests/pages/adminPage.js:48]: Consider the use of template literals

Comment (puneet0191) [reply pmm-app/tests/pages/adminPage.js]: Thanks, I have addressed your comments.

**Pull request merged**